packages/compass-import-export/src/modules/export.js

```diff
@@ -489,62 +487,72 @@ export const startExport = () => {
             projection[parts.slice(0, index + 1).join('.')]
         )
     );
-    let formatter;
-    if (exportData.fileType === 'csv') {
-      formatter = createCSVFormatter({ columns });
-    } else {
-      formatter = createJSONFormatter();
-    }
-
     const dest = fs.createWriteStream(exportData.fileName);
```

Comment:
What would happen if `createWriteStream` and  `dataService.fetch` throws? does Compass crash?